tests/Application/Commands/SendStatisticsTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MatchBot\Tests\Application\Commands;
+
+use Aws\CloudWatch\CloudWatchClient;
+use MatchBot\Application\Commands\SendStatistics;
+use MatchBot\Application\Environment;
+use MatchBot\Domain\DonationRepository;
+use MatchBot\Tests\TestCase;
+use Psr\Log\NullLogger;
+use Symfony\Component\Clock\MockClock;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\Lock\LockFactory;
+
+class SendStatisticsTest extends TestCase
+{
+    private \DateTimeImmutable $startOfMockClocksCurrentMinute;
+
+    public function setUp(): void
+    {
+        $this->startOfMockClocksCurrentMinute = new \DateTimeImmutable('@1727866800');
```

Comment:
I might make another PR when I have time just to use a more readable date/time format here and in the MockClock constructor - there's no way to see at a glance that this is the start of a minute but I do see that its two seconds before the clock is set to so is reasonable.